### DISCUSSION TITLE
How to export activities?

### DISCUSSION THREAD

**Discussion by fmnamado** [ghostfolio/ghostfolio#2966] [Q&A]
Hello,

As I am able to import transactions, I also would like to export transactions.
It is useful to import from excel, as it is useful to export to other platforms.

This way, ghostfolio could be my central data source.

Thank you.

**Comment by dtslvr** [comment]
Hi @fmnamado

You can export your activities from various places, e.g. check _My Ghostfolio_ → _Export Data_.

**Comment by fmnamado** [comment]
Thank you for your help!

I am sorry, let me rephrase the post.

I would like to export in csv/excel, not in JSON.
I am not sure how JSON would help me into inputting this data to other platforms.

What I would want, would like to export a CSV in the same type as the CSV that gets accepted as import origin.

Example: the same CSV file as the one present in

[ghostfolio](https://github.com/ghostfolio/ghostfolio/tree/main)/[test](https://github.com/ghostfolio/ghostfolio/tree/main/test)/[import](https://github.com/ghostfolio/ghostfolio/tree/main/test/import)
/ok.csv

**Reply by dtslvr** [reply]
Maybe you can use an online _JSON to CSV_ converter like [this](https://retool.com/utilities/convert-json-to-csv) one.
